### PR TITLE
Add image, text and input components

### PR DESCRIPTION
**Pull request by jalajcodes**
Just check `.scss` files. `.css` and `.map` files are automatically generated and don't need to be reviewed.

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -58,6 +58,17 @@ <h1>Quick start</h1>
...
+    </pre>
+            <h4>Fonts and Icons</h4>
+            <p>
+              To use the fonts and icons used in the component library add these tag inside your
+              <code>head</code> tag.
+            </p>
+            <pre>
+    <code class="html">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Fira+Code&family=Poppins:wght@400;500;600&display=swap" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" />
```
Comment (MayankKumar10) [inline docs.html:72]: you can also define css links in root directory so don't have to write them down everywhere

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good to me

File: docs.html (hunk 1 above)
Comment (jalajcodes) [reply docs.html:72]: will do that

File: docs.html [hunk 2]
```diff
@@ -651,6 +652,135 @@ <h4>Card with Overlay</h4>
...
+            </div>
+            <pre>
+              <code class="html">
+<p class="gray-4">A quick brown fox jumps over the lazy dog</p>
+<p class="gray-3">A quick brown fox jumps over the lazy dog</p>
+<p class="gray-2">A quick brown fox jumps over the lazy dog</p>
+<p class="gray-1">A quick brown fox jumps over the lazy dog</p>
+              </code>
+            </pre>
+
```
Comment (anik31) [inline docs.html:769]: maintain code indentation

**Review by anik31 — APPROVED** [review]
Everything is good. UI is very nice.

File: docs.html (hunk 2 above)
Comment (jalajcodes) [reply docs.html:769]: Actually if I indent this code, it will come indented in the UI.

**Pull request merged**